lib/issue_exporter.rb

```diff
@@ -3,7 +3,11 @@
 require 'English'
 require 'shellwords'
 require 'json'
+require 'issue_exporter/github'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.